showcase/src/assets/examples/app-data/dropdown.ts

```diff
@@ -1,12 +1,33 @@
 import { Component, Inject, OnInit } from "@angular/core";
-import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+import {
+	STARK_APP_METADATA,
+	STARK_USER_SERVICE,
+	StarkApplicationMetadata,
+	StarkUser,
+	StarkUserService
+} from "@nationalbankbelgium/stark-core";
+import * as moment from "moment";
+import { filter } from "rxjs/operators";
 
 @Component({
 	selector: "demo-app-data",
 	templateUrl: "./demo-app-data.component.html"
```

Comment:
Can you add the `styleUrls` here? Because this example also includes some CSS ;)